src/graphics/game-layout/components/RunInfo.vue

```diff
@@ -1,66 +1,38 @@
 <template>
-  <div
-    class="Flex"
-    :style="{
-      height: '100%',
-      width: '100%',
-      'justify-content': 'center',
-    }"
-  >
+  <div class="Flex runInfoRoot">
     <div
       v-show="runData"
-      ref="RunInfo"
-      class="FlexColumn has-side-lines"
+      class="FlexColumn runDataContainer has-side-lines"
       :style="{
-      'box-sizing': 'border-box',
       'border-right': lineRight ? '5px solid var(--slide-color)' : '5px solid rgba(0,0,0,0)',
       'border-left': lineLeft ? '5px solid var(--slide-color)' : '5px solid rgba(0,0,0,0)',
-      'font-size': '36pt',
-      'font-family': 'Goodlight',
-      'text-align': 'left',
-      // padding: '5px 20px',
       'line-height': lineHeight || 'unset',
-      'white-space': 'normal',
-      'padding-top': '8px',
     }"
     >
       <div
-        class="Flex"
+        class="Flex RunGameParent"
+        ref="RunInfo"
         :style="{
-      width: '100%',
-      height: '75px',
-      'text-align': textAlignCss,
-      position: 'relative',
-      'justify-content': textAlign,
-      'align-items': textAlign,
-    }">
+          'text-align': textAlignCss,
+          'justify-content': textAlign,
+          'align-items': textAlign,
+        }"
+      >
         <div
           v-show="runData && runData.game"
-          class="RunGame"
-          :style="{
-        'font-family': 'Goodlight',
-        'font-size': '20px',
-        'font-weight': 600,
-        'line-height': 1.5,
-        display: 'flex',
-        'max-width': '100%',
-      }">
-          <template v-if="runData && runData.game">
-            {{ runData.game }}
-          </template>
+          class="Flex RunGame"
+        >
+          {{ gameNameUpper }}
         </div>
       </div>
 
       <div
-        class="Flex"
+        class="Flex runInfoExtraContainer"
         :style="{
-      width: '100%',
-      height: '100%',
-      'font-size': '55pt',
-      'text-align': textAlignCss,
-      'justify-content': textAlign,
-      'align-items': 'baseline',
-    }">
+          'text-align': textAlignCss,
```

Comment:
you're using this inline css for more than 1 spot, consider putting in its own css class and not define it inline

Reply:
Can't, it's using vue properties

Reply:
https://www.google.com/search?client=firefox-b-d&q=use+vue+variable+in+css

Reply:
booooo

Reply:
Also remember, we're still on vue 2 https://v2.vuejs.org/v2/guide/class-and-style